app/src/main/java/org/p2p/wallet/bridge/send/BridgeSendInteractor.kt

```diff
@@ -74,7 +74,7 @@ class BridgeSendInteractor(
             currency = token.mintAddress
         )
         return feeRelayerRepository.signTransaction(
-            transaction = signedTransaction.transaction.base58Value.toBase64String(),
+            transaction = signedTransaction.transaction.base58Value.toBase64Instance(),
```

Comment:
i change it to `signedTransaction.transaction.decodeToBytes().toBase64Instance()`